.gitignore

```diff
@@ -149,4 +149,7 @@ functions/node_modules
 functions/src
 functions/swagger.json
 functions/yarn.lock
-yarn.lock
\ No newline at end of file
+yarn.lock
+
+# prettier
+.prettierignore
```

Comment:
remove this line since it doesn't relate to your changes - if you want to make this change (which I encourage!) you can open a separate PR

Reply:
Okay, I did this to avoid some conflicts I got when I saved the file but I can remove it now

Reply:
@davydocsurg any update on removing these prettier lines? let's focus on this in another PR

Reply:
Done